react-common/components/controls/Input.tsx

```diff
@@ -137,7 +141,7 @@ export const Input = (props: InputProps) => {
             onBlur(value);
         }
         if (!preserveValueOnBlur) {
-            setValue(undefined);
+            setValue("");
```

Comment:
Just for a sanity check, is this change just to stay consistent with the string type? Like changing this to the empty string should still be falsey and not cause problems, right?